doc/source/getting_started/intro_tutorials/04_plotting.rst

```diff
@@ -90,11 +100,11 @@ method. Hence, the :meth:`~DataFrame.plot` method works on both ``Series`` and
 
 I want to visually compare the :math:`NO_2` values measured in London versus Paris.
 
-.. ipython:: python
+.. ipython::
 
     @savefig 04_airqual_scatter.png
-    air_quality.plot.scatter(x="station_london", y="station_paris", alpha=0.5)
-
+    In [1]: air_quality.plot.scatter(x="station_london", y="station_paris", alpha=0.5)
```

Comment:
why are you not letting these run with python mode. this is not what we do elsewhere.

Reply:
Hi @jreback, if I run them in Python mode it displays as below with the extra Out[4] line, which is why I didn't run them in Python mode. I just thought it looked cleaner that way as the Out line doesn't add value.

![Screenshot 2022-04-10 at 19-09-27 How do I create plots in pandas — pandas 1 5 0 dev0 667 gcb426b57ae dirty documentation](https://user-images.githubusercontent.com/20362314/162633357-2d0f4fc5-818c-402a-9519-f269b851090e.png)

If we wanted to get rid of this out line while still running in Python mode I can add semi colons to the end of each .plot() line as demonstrated below. Which of these would you prefer me to do?

![Screenshot 2022-04-10 at 19-12-08 How do I create plots in pandas — pandas 1 5 0 dev0 667 gcb426b57ae dirty documentation](https://user-images.githubusercontent.com/20362314/162633464-c27f15a7-4202-4cec-b299-b085bff159b1.png)

Reply:
@jreback thanks for your comment, please can you explain what you mean by "non-standard"? One of the issues of letting it run in Python mode is the command numbering in each cell, which is why, after some experimentation, it seemed to work better excluding it. Many thanks for your help